ostd/src/task/mod.rs

```diff
@@ -2,15 +2,16 @@
 
 //! Tasks are the unit of code execution.
 
-mod priority;
 mod processor;
 mod scheduler;
 #[allow(clippy::module_inception)]
 mod task;
 
 pub use self::{
-    priority::Priority,
-    processor::{current_task, disable_preempt, preempt, schedule, DisablePreemptGuard},
-    scheduler::{add_task, set_scheduler, FifoScheduler, Scheduler},
-    task::{Task, TaskAdapter, TaskContextApi, TaskOptions, TaskStatus},
+    processor::{current_task, disable_preempt, DisablePreemptGuard},
+    scheduler::{
+        add_task, inject_scheduler, schedule, yield_now, EnqueueFlags, FifoScheduler,
+        LocalRunQueue, Scheduler, UpdateFlags, YieldFlags,
```

Comment:
Do not re-export all APIs under the `scheduler` module. Instead, expose the `scheduler` module and re-export only the frequently-used API from the `scheduler` module. Since no APIs under the `scheduler` module are expected to be used frequently, I suggest re-exporting none of them.

Reply:
I'll fix this as advised.